.github/workflows/dependency-review.yml

```
@@ -0,0 +1,14 @@
+name: 'Dependency Review'
+on: [pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  dependency-review:
```

```suggestion
  validate:
```
Changing the job name to validate will make it match our branch protection rules for the repo - we should also consider if/how we might de-conflict this dep review with `.github/workflows/scan-cves.yml` which does similar things (not license checks) but more manually.  We could leave both running to see which output we like better though over time (`grype` has gotten better but had a lot of false positives when we first started).